roofit/roostats/inc/RooStats/SimpleLikelihoodRatioTestStat.h

```diff
@@ -63,11 +60,13 @@ namespace RooStats {
          fDetailedOutputEnabled = false;
          fDetailedOutput = NULL;
 
-    fReuseNll=kFALSE ;
-    fNllNull=NULL ;
-    fNllAlt=NULL ;
+         fReuseNll=kFALSE ;
+         fNllNull=NULL ;
+         fNllAlt=NULL ;
       }
-      //__________________________________
+
+      /// Takes null and alternate parameters from values in nullParameters
```

Comment:
/// Constructor passing PDF for the null and the alternate models and 
///  the corresponding model parameters